osquery/logger/plugins/logger_plugin_util.h

```diff
@@ -0,0 +1,151 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
```

Comment:
I would vote to move this to `osquery/logger/buffer.cpp` or something. I like having only plugins in `plugins` and the name is pretty long as it is now.